browser/src/control/Control.NotebookbarCalc.js

```diff
@@ -986,6 +986,30 @@ L.Control.NotebookbarCalc = L.Control.NotebookbarWriter.extend({
 
 	getLayoutTab: function() {
 		var content = [
+			{
+				'id': 'Data-OrientationMenu:MenuOrientation',
+				'class': 'unoData-OrientationMenu',
```

Comment:
do we use that class at all? I don't see.

it should be added only when really necessary (exceptions), also I think it was `cssClass` not `class` property
can we remove it? also in other items to not spread it